snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -543,7 +534,7 @@ def prepare_build(self, force=False):
         self.makedirs()
         # Stage packages are fetched and unpacked in the pull step, but we'll
         # unpack again here just in case the build step has been cleaned.
-        self._unpack_stage_packages()
+        self._install_stage_packages()
```

Comment:
This might not be needed anymore for v2... we need to cleanup snapcraft.internal.lifecycle._runner when we have time.